frontend/src/concepts/notebooks/StartNotebookModal.tsx

```diff
@@ -0,0 +1,312 @@
+          </Alert>
+        </StackItem>
+      );
+    }
+
+    if (!spawnStatus && inProgress) {
+      return (
+        <StackItem>
+          <Content>
+            Depending on the size and resources requested, this can take several minutes.
+          </Content>
+        </StackItem>
+      );
+    }
+
+    return null;
+  };
+
+  const renderLogs = () => (
+    <Panel style={{ overflowY: 'auto', height: '100%', padding: SmallSpacerSize.var }}>
+      <PanelMain>
+        {(isStopped || isRunning) && events.length === 0 ? (
+          <span style={{ color: DisabledColor.var }}>There are no recent events.</span>
+        ) : (
+          <List isPlain isBordered data-id="event-logs">
+            {events
+              .toSorted(
+                (a, b) =>
+                  new Date(getEventTimestamp(b)).getTime() -
+                  new Date(getEventTimestamp(a)).getTime(),
+              )
+              .map((event, index) => (
+                <ListItem key={`notebook-event-${event.metadata.uid ?? index}`}>
+                  {getEventFullMessage(event)}
+                </ListItem>
+              ))}
+            <ListItem>Server requested</ListItem>
+          </List>
+        )}
+      </PanelMain>
+    </Panel>
+  );
+
+  const renderProgress = () => (
+    <Flex
+      direction={{ default: 'column' }}
+      gap={{ default: 'gapMd' }}
+      style={{ overflowY: 'auto', height: '100%' }}
+    >
+      <FlexItem>
+        <Flex gap={{ default: 'gapSm' }} flexWrap={{ default: 'nowrap' }}>
+          <FlexItem>
+            <InfoCircleIcon
+              style={{
+                color: InfoColor.var,
+                fontSize: InfoIconSize.var,
+                paddingTop: ExtraSmallSpacerSize.var,
+              }}
+            />
+          </FlexItem>
+          <FlexItem>
+            Steps may repeat or occur in any order, depending on the workbench’s priority in the
+            queue and current resource availability.
+          </FlexItem>
+        </Flex>
+      </FlexItem>
+      <FlexItem flex={{ default: 'flex_1' }} style={{ overflowY: 'auto' }}>
+        <ProgressStepper isVertical data-testid="notebook-startup-steps">
+          {notebookProgress.map((progressStep, i) => (
+            <ProgressStep
+              key={`${progressStep.timestamp}-${i}`}
+              variant={progressVariants[progressStep.status]}
+              aria-label={progressStep.status}
+              id={`${progressStep.timestamp}`}
+              titleId={`${progressStep.timestamp}-title`}
+              data-testid={`step-status-${progressStep.status}`}
+            >
+              {progressStep.step
+                ? ProgressionStepTitles[progressStep.step]
+                : progressStep.description}
+            </ProgressStep>
+          ))}
+        </ProgressStepper>
+      </FlexItem>
+    </Flex>
+  );
+
+  return (
+    <Modal
+      appendTo={document.body}
+      variant={ModalVariant.small}
```

Comment:
`variant="small"`

Reply:
Using the `ModalVariant` enum is better, I left that alone.